match2/commons/match_group_display_matchlist.lua

```diff
@@ -113,8 +113,8 @@ function MatchlistDisplay.Matchlist(props)
 			:node(matchNode)
 	end
 
-	return html.create('div'):addClass('brkts-main')
-		:cssText(config.attached and 'padding-left:0px;padding-right:0px' or nil)
+	return html.create('div'):addClass('brkts-main brkts-main-dev-2')
```

Comment:
Hmm, what's up with this?

Reply:
css changes associated with this pr are gated behind `brkts-main-dev-2`